src/test/java/tests/BaseClass.java

```diff
@@ -0,0 +1,42 @@
+package tests;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Optional;
+import org.testng.annotations.Parameters;
+
+import java.util.logging.Logger;
+
+public class BaseClass {
+    protected WebDriver driver;
+    //protected int WAIT_TIMEOUT_SECONDS;
```

Comment:
please remove commented code

Reply:
Thank you for pointing, please see in the following commit.